torchvision/datasets/utils.py

```diff
@@ -28,27 +28,24 @@
 USER_AGENT = "pytorch/vision"
 
 
-def _urlretrieve(url: str, filename: str, chunk_size: int = 1024) -> None:
```

Comment:
Both this function and `_save_response_content` below had the same functionality, so I factored it out.